src/scipp/plotting/figure3d.py

```diff
@@ -22,7 +22,7 @@ class PlotFigure3d:
     It renders an interactive scene containing a point cloud using `pythreejs`.
     """
     def __init__(self, *, background, cmap, extend, figsize, mask_cmap, nan_color, norm,
-                 pixel_size, show_outline, tick_size, xlabel, ylabel, zlabel):
+                 pixel_size, show_outline, tick_size, xlabel, ylabel, zlabel, camera):
```

Comment:
What types can` camera` be? Only dictionary or something else?

Reply:
Only `dict`. See https://github.com/scipp/scipp/pull/2361/files#diff-5a810919acade02ff1c94e0ddee32a1317e7bf304fffa5eb55744aa1d5e44f8eR92

Reply:
That said, we should add type hints to all the plotting code.